### PR TITLE
fix auth. errors

### PR DESCRIPTION
**Pull request by gaelrehault**

### CODE REVIEW

File: src/deploy/auto_common/Scp.py [hunk 1]
```diff
@@ -28,21 +28,21 @@ def get_file(adress, user, passw, localfile, remotefile):
...
+        client.connect(hostname=adress, username=str(user), password=str(passw))
+        sftp=client.open_sftp()
         sftp.get(remotefile, localfile)
         sftp.close()
-        trans.close()
+        client.close()
 
     @staticmethod
     def put_file(adress, user, passw, localfile, remotefile):
+        #sftp.put(localfile, remotefile)
```
Comment (heydog15) [inline src/deploy/auto_common/Scp.py]: This line is a  commented dupe of line #45.  Is it needed or can it be removed?

File: src/deploy/auto_common/Scp.py [hunk 2]
```diff
@@ -28,21 +28,21 @@ def get_file(adress, user, passw, localfile, remotefile):
...
-        trans = paramiko.Transport((adress, 22))
-        trans.connect(username=user, password=passw)
-        sftp = paramiko.SFTPClient.from_transport(trans)
+        client.connect(hostname=adress, username=str(user), password=str(passw))
+        sftp=client.open_sftp()
         sftp.put(localfile, remotefile)
         sftp.close()
-        trans.close()
+        client.close()
+
```
Comment (heydog15) [inline src/deploy/auto_common/Scp.py]: Is this line needed?

File: src/deploy/auto_common/Scp.py (hunk 1 above)
Comment (gaelrehault) [reply src/deploy/auto_common/Scp.py]: removed

File: src/deploy/auto_common/Scp.py (hunk 2 above)
Comment (gaelrehault) [reply src/deploy/auto_common/Scp.py]: yes.

**Comment by heydog15** [comment]
Looks good

**Pull request merged**